content/en/docs/collector/howto-receiver-trace.md

```diff
@@ -0,0 +1,1912 @@
+2022-03-03T11:19:50.788-0600    info    service/collector.go:239        Starting dev-otelcol... {"Version": "1.0.0", "NumCPU": 12}
+2022-03-03T11:19:50.788-0600    info    service/collector.go:135        Everything is ready. Begin running and processing data.
+2022-03-21T15:19:51.717-0500	info	jaegerexporter@v0.46.0/exporter.go:186	State of the connection with the Jaeger Collector backend	{"kind": "exporter", "name": "jaeger", "state": "READY"}
+2022-03-03T11:20:51.783-0600    info    tailtracer/trace-receiver.go:23  I should start processing traces now!   {"kind": "receiver", "name": "tailtracer"}
+```
+
+  
+
+Look for the log line for "builder/receivers_builder.go:68 Receiver is starting... {"kind": "receiver", "name": "tailtracer"}", you can see that the Collector found the settings for the `tailtracer` receiver within the `traces` pipeline and is now instantiating it and starting it given that 1 minute after the Collector has started, you can see the info line we added to the `ticker` function within the `Start()` method.
+
+Now, go ahead and press `ctrl+c` in your Collector's terminal so you want watch the shutdown process happening. Here is what the output should look like:
+
+```cmd
+^C2022-03-03T11:20:14.652-0600  info    service/collector.go:166        Received signal from OS {"signal": "interrupt"}
+2022-03-03T11:20:14.652-0600    info    service/collector.go:255        Starting shutdown...
+2022-03-03T11:20:14.652-0600    info    service/service.go:121  Stopping receivers...
+2022-03-03T11:20:14.653-0600    info    tailtracer/trace-receiver.go:29  I am done and ready to shutdown!        {"kind": "receiver", "name": "tailtracer"}
+2022-03-03T11:20:14.653-0600    info    service/service.go:126  Stopping processors...
+2022-03-03T11:20:14.653-0600    info    builder/pipelines_builder.go:73 Pipeline is shutting down...    {"name": "pipeline", "name": "traces"}
+2022-03-03T11:20:14.653-0600    info    builder/pipelines_builder.go:77 Pipeline is shutdown.   {"name": "pipeline", "name": "traces"}
+2022-03-03T11:20:14.653-0600    info    service/service.go:131  Stopping exporters...
+2022-03-03T11:20:14.653-0600    info    service/service.go:136  Stopping extensions...
+2022-03-03T11:20:14.653-0600    info    service/collector.go:273        Shutdown complete.
+```
+  
+
+As you can see there is an info log line for the `tailtracer` receiver which means the component is responding correctly to the `Shutdown()` event.
+In the next section you will learn more about the OTel Trace data model so the `tailtracer` receiver can finally generate traces!
+
+## The Collector's Trace Data Model
+
+You might be familiar with OTel traces by using the SDKs and instrumenting an application so you can see and evaluate your traces within a distributed tracing backend like Jaeger. 
+
+Here is what a trace looks like in Jaeger:
+
+![Jaeger trace](images/Jaeger.jpeg)
```

Comment:
For all images, please add them to `/static/img/docs/<newDir>/` and reference them as `(/img/docs/<newDir>/<image>`.

Reply:
done